test_package/CMakeLists.txt

```diff
@@ -6,11 +6,7 @@ set(CMAKE_VERBOSE_MAKEFILE TRUE)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
-if(NOT MSVC)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-else()
-  add_definitions(-D_WIN32_WINNT=0x600)
-endif()
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
This is only available in CMake 3.1+
Please raise the cmake_minimum_required version to 3.1